src/rpft/rapidpro/models/routers.py

```diff
@@ -371,8 +373,6 @@ def from_dict(data, exits):
         return RandomRouter(result_name, categories)
 
     def add_choice(self, category_name=None, destination_uuid=None):
-        if not category_name:
-            category = f"Bucket {len(self.categories) + 2}"
```

Comment:
@geoo89 this was flagged as redundant; is it ok to delete?

Reply:
I think this is another bug: `category` in line 375 should be `category_name`.